README.md

```diff
@@ -2,14 +2,20 @@
 
 # 🚀 gdx-liftoff 🚀
 
+## 🎛 UI Redesign 🎛
+
 A setup tool for [libGDX](https://libgdx.com/) Gradle projects.
 
 [![Screenshot of gdx-liftoff](.github/screenshot.png)](https://github.com/libgdx/gdx-liftoff/releases)
 
 <h1 style="text-align: center;">
     📥
-    <strong><a href="https://github.com/libgdx/gdx-liftoff/releases">DOWNLOAD</a></strong>
+    <strong><a href="https://github.com/libgdx/gdx-liftoff/releases">DOWNLOAD STABLE</a></strong>
     📥
+<br>
+    🎉
```

Comment:
This is Tommy reviewing my own changes, but I just need to remember to remove the preview link when we have a release.